src/templates/show-templates-select.html

```diff
@@ -40,7 +40,7 @@
               <a href='?tab=3&mode=edit&templateid={{ template.id }}'><span class='p-2 rounded hl-hover-gray'><i class='fas fa-pencil-alt clickable link-like'></i></span></a>
             {% endif %}
               {# clickable toggle pin button #}
-              <span title='{{ 'Toggle pin (add to create button menu)'|trans }}' data-action='toggle-pin' data-id='{{ template.id }}' id='toggle-pin-icon-div' class='{{ template.is_pinned ? 'bgnd-gray' : 'hl-hover-gray' }} p-1 rounded'><i id='toggle-pin-icon' class='fas fa-thumbtack {{ not template.is_pinned ? 'color-weak' }} fa-fw'></i></span>
+              <span title='{{ 'Toggle pin (add to create button menu)'|trans }}' data-action='toggle-pin' data-id='{{ template.id }}' class='{{ template.is_pinned ? 'bgnd-gray' : 'hl-hover-gray' }} p-1 rounded'><i data-toggle-pin-icon='' class='fas fa-thumbtack {{ not template.is_pinned ? 'color-weak' }} fa-fw'></i></span>
```

Comment:
I understand that this was changed to remove the duplicated "id", but the new code breaks the behavior for Experiments/Items.